authorize.py

```diff
@@ -136,16 +65,12 @@ def create_user(event, context, mlh = False):
     except KeyError:
        return config.add_cors_headers({"statusCode":400, "body":"No email provided!"})
 
-    #MLH users don't have a password with us.
-    if mlh == True:
-        event['password'] = "defacto"
-
     if 'password' not in event:
        return config.add_cors_headers({"statusCode":400, "body":"No password provided"})
 
     u_email = event['email']
     password = event['password']
-    password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+    password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt(rounds=8))
```

Comment:
why 8? might wanna put this in config so we can change rounds when needed

Reply:
Tru, tho we'd have to invalidate people then, I think